src/redux/dispatchers/token.ts

```diff
@@ -7,18 +6,22 @@ import {
   approve as approval,
   transfer as transferral,
 } from '../action-creators/token'
+import { resetTokenApprove } from '../action-creators/token/approve'
+import { resetTokenTransfer } from '../action-creators/token/transfer'
 
-const approve = async (address:string, amount:Nos, from:string): Promise<TransactionReceipt> =>
+const approve = async (address:string, amount:Nos, from:string): Promise<{ [key: string]: string }> =>
```

Comment:
i think i exported a `keyed` interface in comp.js that is this -- if i didnt you can make one here rather than repeatedly define this shape